Week2/Section2_w2/MySpotify/Song.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Song.h"
+#include <string>
+
+Song::Song()
+{
+	SongTitle = "";
+	ArtistName = "";
+	Duration = 0;
+}
+
+Song::Song(const std::string NewSongTitle, const std::string NewArtistName, const int NewDuration)
+{
+	SongTitle = NewSongTitle;
+	ArtistName = NewArtistName;
+	Duration = NewDuration;
+}
+
+std::string Song::GetFormatedDuration()
+{
+	std::string Minutes = std::to_string(Duration / 100);
+	std::string Seconds = std::to_string(Duration - (Duration / 100) * 100);
+	return Minutes + ":" + Seconds;
+}
+
+bool Song::IsValid()
+{
+	if (isupper(SongTitle[0]))
+	{
+		return true; //if the song name starts with an uppercase letter, is valid
+	}
+	else
+	{
+		return false;
```

Comment:
This code works perfectly fine, but it is a bit complex for how trivial it is. Instead of writing
```
if (bCondition)
{
    return true;
}
else
{
    return false;
}
```
you could just write
```
return bCondition;
```
So, in this case, you could just write:
```
return isupper(SongTitle[0]);
```
That way your function is clearer and easier to read, and you saved 7 lines for such a simple function.

Reply:
Great advice, i'll implement it